### ISSUE TITLE
pio blink example incorrect frequency

### ISSUE THREAD

**Issue by smist09** [raspberrypi/pico-sdk#360]
The line in the C program is:

    pio->txf[sm] = clock_get_hz(clk_sys) / 2 * freq;

Should be:

    pio->txf[sm] = clock_get_hz(clk_sys) / freq;

**Comment by lurch** [comment]
Duplicate of https://github.com/raspberrypi/pico-examples/issues/88 ?

**Comment by smist09** [comment]
You are correct, though notice in the other issue should mention removing the 2.
